tripleblindmarket/private-kit
# [SAF-848] update cursor caching

#### Description:

Added the ability to store checksum as cacheKey in async storage when using new cursor file version.

#### How to test:

Since this is a new feature on safe places backend, there is currently a fallback in place for old version of HA cursor files. 

In version 1.1 we should see a checksum field when fetching cursor file. Checksum helps us determine whether the contents of fetched page has changed which in turn reduces unnecessary fetching of data and helps us keep locally stored data updated when HA publishes updates. 

This feature currently lives in test(staging) HA so to test this we need to subscribe to staging HA, wait for our app to fetch data and then update one or more pages(change hashes) and then wait for automatic update or force Cursor update by resubscribing
